docs/data/material/components/button-group/LoadingButtonGroup.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import ButtonGroup from '@mui/material/ButtonGroup';
+import Button from '@mui/material/Button';
+import LoadingButton from '@mui/lab/LoadingButton';
+import SaveIcon from '@mui/icons-material/Save';
+
+export default function LoadingButtonGroup() {
+  return (
+    <ButtonGroup variant="contained" aria-label="loading button group">
```

Comment:
```suggestion
    <ButtonGroup variant="outlined" aria-label="loading button group">
```

Let's use the outlined variant